packages/atomic/src/components/commerce/atomic-commerce-layout/e2e/atomic-commerce-layout.e2e.ts

```diff
@@ -0,0 +1,12 @@
+import {test, expect} from './fixture';
```

Comment:
These tests seem out of scope for this PR. Not a huge deal, but I wonder if they were included in the PR accidentally.